test/SpagoSpec.hs

```diff
@@ -185,12 +246,23 @@ spec = around_ setup $ do
       writeTextFile "packages.dhall" "let pkgs = ./packagesBase.dhall in pkgs // { spago = { dependencies = [\"prelude\"], repo = \"https://github.com/purescript/spago.git\", version = \"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\" }}"
       spago ["install", "spago"] >>= shouldBeFailure
 
-    it "Spago should be able to update dependencies in an alternative config" $ do
+    describe "Spago should be able to update dependencies in an alternative config" $ do
 
-      spago ["init"] >>= shouldBeSuccess
-      writeTextFile "alternative1.dhall" "./spago.dhall // {dependencies = [\"prelude\"]}"
-      spago ["-x", "alternative1.dhall", "install", "simple-json"] >>= shouldBeSuccess
-      checkFixture "alternative1.dhall"
+      it "... alternative1.dhall" $ do
+
+        spago ["init"] >>= shouldBeSuccess
+        writeTextFile "alternative1.dhall" "./spago.dhall // {dependencies = [\"prelude\"]}"
+        spago ["-x", "alternative1.dhall", "install", "simple-json"] >>= shouldBeSuccess
+        checkFixture "alternative1.dhall"
+
+      it "... alternative3-insane.dhall qw" $ do
```

Comment:
`qw`?

Reply:
Left over from when I was debugging this. When I run `stack test --fast --ta="--match='some description'"` I get a parsing failure because it gets interpreted as `stack --fast --ta="--match='some' description"`, which fails because it has no idea what CLI arg `description` refers to. So, adding `qw` was an easier way to run specific tests.